configuration/etl_ddl/tb_screening/tb_screening.sql

```diff
@@ -0,0 +1,55 @@
+DELIMITER //
```

Comment:
@Michaelndula please maintain this as a create table statement without the stored precedure